challenge_10/rack_response.rb

```diff
@@ -0,0 +1,93 @@
+### Require dependencies
+require 'cgi'
+require 'rack/handler/puma'
+require 'rack'
+require 'uri'
```

Comment:
My bad for [missing this](https://github.com/adrianna-chang-shopify/learning-project/pull/8/commits/db2c5efdd30369936e3e32e7e2d7dc5bdaa835f2#r601302364) in #8, but we’re no longer using `URI` because [`Rack::QueryParser`](https://github.com/rack/rack/blob/v2.2.2/lib/rack/query_parser.rb) does all the hard work for us.